src/components/AppStyle.tsx

```diff
@@ -60,6 +60,9 @@ const theme = {
       borders: palette.gray20,
       modalOverlay: "rgba(0, 0, 0, 0.7)",
     },
+    viewport: {
+      background: "#f7f7f7",
```

Comment:
nit: but everything else in here is defined by a constant

Reply:
Fixed, thank you!